### DISCUSSION TITLE
Star the project for updates!

### DISCUSSION THREAD

**Discussion by deepak-ghimire** [dearhive/dearflip-js-flipbook#60] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cperera-cell** [comment]
Hi i need to get the cover (first page) to open when the flip book opens but it works in desktop but not in mobile.In mobile the third page opens at first. These are my settings.

<img width="1078" height="721" alt="Screenshot 2026-02-11 152222" src="https://github.com/user-attachments/assets/5de83a99-11e6-4928-aed0-67b90c29d15e" />

**Reply by deepak-ghimire** [reply]
Hi,
Third page cannot open unless third page is set as start page.

Can you share the link to your page? We will have a look at it.

**Reply by cperera-cell** [reply]
Hi, This is the link
https://lacasastudio.it/la-casa-studio-immobiliare-brochure/

**Reply by deepak-ghimire** [reply]
When I open the link in mobile it opens the first page. Can you share a video of the flipbook starting at page 3?

**Reply by cperera-cell** [reply]
https://github.com/user-attachments/assets/e89d0817-dd9a-4c5b-a7a8-bf72dbba1a37

I tried even on iphone and android both are the same

**Reply by deepak-ghimire** [reply]
The link https://lacasastudio.it/la-casa-studio-immobiliare-brochure/
and the link you are using in the video are a bit different, In the link you are using it clearly mentions page 3. that is direct link to the page, obviously it will open page 3. 

Open first page
https://lacasastudio.it/la-casa-studio-immobiliare-brochure/ 

Opens page 2
https://lacasastudio.it/la-casa-studio-immobiliare-brochure/#flipbook-df_41951/2/

Opens page 10
https://lacasastudio.it/la-casa-studio-immobiliare-brochure/#flipbook-df_41951/10/

so on.
